pkgs/development/python-modules/boxx/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, xvfb-run
+, matplotlib
+, scikitimage
+, numpy
+, pandas
+, imageio
+, snakeviz
+, fn
+, pyopengl
+, seaborn
+, pytorch
+, torchvision
+}:
+
+buildPythonPackage rec {
+  pname = "boxx";
+  version = "0.9.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Mc6R6ruUVhFs2D0CTJsAiM9aGOusS973hRS5r2kQsy4=";
+  };
+
+  propagatedBuildInputs = [
+    matplotlib
+    scikitimage
+    numpy
+    pandas
+    imageio
+    snakeviz
+    fn
+    pyopengl
+    seaborn
+  ];
+
+  pythonImportsCheck = [ "boxx" ];
+  checkInputs = [
+    xvfb-run
+    pytorch
+    torchvision
+  ];
+
+  checkPhase = ''
+    xvfb-run python -m unittest
```

Comment:
I am not sure if we want to use ``${python.interpreter}`` here, too. If it does not work on the first or second try, we can ignore it.

Reply:
It seems that in this case doesn't matter

boxx depends indirectly on protobuf and it's not supported on pypy for example

If in the future that library works again on pypy it's less things to change and less things to break